drivers/virtualbox/virtualbox.go

```diff
@@ -81,6 +82,11 @@ func GetCreateFlags() []cli.Flag {
 			Usage:  "The URL of the boot2docker image. Defaults to the latest available version",
 			Value:  "",
 		},
+		cli.StringFlag{
+			Name:  "virtualbox-import-boot2docker-vm",
+			Usage: "The name of a Boot2Docker VM to import",
+			Value: "",
```

Comment:
I keep thinking that maybe we should default this value to `boot2docker-vm`, since that's by far the most common name and I have a feeling users will run this flag without args expecting it to "just work".